jupyter_book/__init__.py

```diff
@@ -40,6 +44,7 @@ def setup(app):
     # Transforms
     app.add_post_transform(SwapTableOfContents)
 
```

Comment:
This comment seems a bit out of place since there's nothing underneath it....

Reply:
you mean `# Transforms`? Should we change it to `#Post transforms`?  I think we were using this comment format to list different kinds of additions to the app.

Reply:
no just underneath this line there is `# Extensions` and then it goes right to the "return" statment